macros/src/main/scala/gestalt/macros/DefMacros.scala

```diff
@@ -20,6 +20,26 @@ object plusObject {
         toolbox.Lit(null)
     }
   }
+
+  inline def varargs(tped: Int*): Int = meta {
+    val q"$items: $_" = tped
+    items match {
+      case toolbox.SeqLiteral(items: Seq[toolbox.Tree]) =>
+        items.reduceLeft((a, b) => q"$a + $b")
+      case _ =>
+        q"$items.reduce((a:Int,b:Int)=> a + b)"
+    }
+  }
```

Comment:
As a macro author, I'm thinking if the macro system can do the heavy lifting so that `tped` is a list of trees, so that the irregularities are not exposed to macro authors?